ui/src/components/Document/DocumentTitle.js

```diff
@@ -1,11 +1,22 @@
 import { DocumentEdition } from '@components/Document';
 import React from 'react';
+import Truncate from 'react-truncate';
 
-export const DocumentTitle = ({ document }) => {
+export const DocumentTitle = ({ document, short, truncate }) => {
```

Comment:
IMHO, it is tricky to play with the args like this. If I want a `short`+`truncate`? :)

I don't have great suggestions... what about something like this? (pseudo-code)
```
const DocumentTitle = ({ document, options={short: false, truncate:false } }) => {
    const title = short ? document.metadata.title : `{document.metadata.title} (<DocumentEdition document={document} /> {document.metadata.publication_year})`;
    const cmp = <div className="document-title">{title}</div>;
    return truncate ? <Truncate lines={2} ellipsis={'... '}>{cmp}</Truncate> : cmp;
```